File: hw/sanchez/u3/hw18OOPConceptsZoo/ZooPython/model/slaughter_house.py
class SlaughterHouse:
    def __init__(self, id: int, name: str, location: str):
        self.id = id
        self.name = name
        self.location = location
    
    def __str__(self):
        return f"SlaughterHouse(id={self.id}, name='{self.name}', location='{self.location}')"

File: hw/sanchez/u3/hw18OOPConceptsZoo/ZooJavaScript/model/SlaughterHouse.js
class SlaughterHouse {
  constructor(id, name, location) {
    this.id = id;
    this.name = name;
    this.location = location;
  }

  toJSON() {
    return {
      id: this.id,
      name: this.name,
      location: this.location
    };
  }
}

module.exports = SlaughterHouse;